modules/indexing/src/test/java/org/apache/ignite/internal/processors/query/stat/StatisticsViewsTest.java

```diff
@@ -56,21 +52,13 @@ public abstract class StatisticsViewsTest extends StatisticsAbstractTest {
      */
     @Test
     public void testConfigurationView() throws Exception {
-        collectStatistics(StatisticsType.GLOBAL, SMALL_TARGET);
-
-        ObjectStatisticsImpl smallStat = (ObjectStatisticsImpl)statisticsMgr(0).getLocalStatistics(SMALL_KEY);
-
-        long aVer = smallStat.columnStatistics("A").version();
-        long bVer = smallStat.columnStatistics("B").version();
-        long cVer = smallStat.columnStatistics("C").version();
-
         List<List<Object>> config = Arrays.asList(
-            Arrays.asList(SCHEMA, "TABLE", "SMALL", "A", (byte)15, null, null, null, null, aVer),
-            Arrays.asList(SCHEMA, "TABLE", "SMALL", "B", (byte)15, null, null, null, null, bVer),
-            Arrays.asList(SCHEMA, "TABLE", "SMALL", "C", (byte)15, null, null, null, null, cVer)
+            Arrays.asList(SCHEMA, "TABLE", "SMALL", "A", (byte)15, null, null, null, null, 1L),
```

Comment:
If this test will be executed after testEnforceStatisticValues - there will be undeleted data and test will fail.